src/devices/Pcx857x/Pcx857x.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems> <!--Disabling default items so samples source won't get build by the main library-->
+    <RootNamespace>Iot.Device.Pcx857x</RootNamespace>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--
+      Temporarily making a project reference until the daily has IGpioController
+      <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" /> 
```

Comment:
This shouldn't be necessary any longer, do you mind removing this workaround from the bindings that are using it please?